models/database.go

```diff
@@ -260,6 +260,16 @@ var databaseSchema = [][]string{
 			ALTER COLUMN rds_basic_metrics_disabled DROP DEFAULT,
 			ALTER COLUMN rds_enhanced_metrics_disabled DROP DEFAULT`,
 	},
+
+	13: {
+		`ALTER TABLE services
+			DROP CONSTRAINT address_socket_check`,
+
+		`ALTER TABLE services
+			ADD CONSTRAINT address_socket_check CHECK (
+				(address IS NOT NULL AND socket IS NULL) OR (address IS NULL AND socket IS NOT NULL) OR (address IS NULL AND socket IS NULL)
```

Comment:
maybe for the third condition we should add service type, what do you think?

Reply:
Well, I think, it is not a very good solution to duplicate full validation in DB.
Such a concept can lead to:
-  the creation of validation stored procedures as constraints - and keeping them in sync it with models;
-  overcomplicate migration without the right instruments to apply/handle them.

And I guess soon the external services will also have host/socket connection to store.

Do you have any strong pros arguments about this?